spec/fortnox/api/repositories/invoice_spec.rb

```diff
@@ -1,10 +1,25 @@
 require 'spec_helper'
 require 'fortnox/api/repositories/context'
 require 'fortnox/api/repositories/invoice'
-require 'fortnox/api/models/invoice'
+require 'fortnox/api/repositories/examples/all_examples'
+require 'fortnox/api/repositories/examples/find_examples'
+require 'fortnox/api/repositories/examples/save_examples'
 
 describe Fortnox::API::Repository::Invoice do
   include_context 'repository context'
 
-  it_behaves_like 'repositories', Fortnox::API::Model::Invoice
+  it_behaves_like 'repositories'
+
+  include_examples '.all'
+
+  include_examples '.find', :document_number
+
+  include_examples '.save', :comments, 'Comments' do
+    let( :attribute_value ){ 'An invoice comment' }
+    let( :updated_attribute_value ){ 'An updated invoice comment' }
+    let( :valid_model ) do
```

Comment:
As I said, I created a shared example, but you created the test in the first place: https://github.com/my-codeworks/fortnox-api/commit/53bb403869cf8cb41c18256ea71f5816f0a7c611

So, as it was you who initially wrote the test, I was hoping that you could explain what the test actually test.
```
before do
  # Should not make an API request in test!
  expect( repository ).not_to receive( :save_new )
  expect( repository ).not_to receive( :update_existing )
end

specify{ expect( repository.save( model )).to eql( true ) }
```
I assume that `# Should not make an API request in test!` was before we used VCR? If I read the test correctly, we just check that when calling `repository.save( model )` it returns `true`, right? I don't think we need to check that anymore since we use VCR. If `save` returns `false`, something is wrong with the request and VCR will nofity.